Common/MathUtils/src/Chebyshev3D.cxx

```diff
@@ -507,15 +507,15 @@ Int_t Chebyshev3D::chebyshevFit(int dmOut)
 {
   // prepare paramaterization of 3D function for dmOut-th dimension
   int maxDim = 0;
-  for (int i = 0; i < 3; i++) {
-    if (maxDim < mNumberOfPoints[i]) {
-      maxDim = mNumberOfPoints[i];
+  for (int mNumberOfPoint : mNumberOfPoints) {
+    if (maxDim < mNumberOfPoint) {
```

Comment:
I disagree. The refactoring makes sure there are no indices involved, which eliminates a large fraction of the "out of bounds" mistakes. Anyways, unless I convinced you with this argument, I guess we can switch it off for now and rediscuss.